packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/internal/span/CustomStyleSpan.kt

```diff
@@ -76,7 +77,11 @@ public class CustomStyleSpan(
         fontFeatureSettings = fontFeatureSettingsParam
         setTypeface(typeface)
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-          fontVariationSettings = fontVariationSettingsParam
+          try {
+            fontVariationSettings = fontVariationSettingsParam
+          } catch (e: Exception) {
+            // Do nothing
+          }
```

Comment:
We do nothing to fallback to default font. Do you think we need some kind of validation? If yes, do you think it's better to be done on JS side or native side?

Reply:
We are wanting to add more sophisticated parsing in native for CSS syntax later, where I think we could check for some of these, but it seems like current design fully relies on Android Paint for understanding of string contents.

No-op on invalid specified style is fine, but we should catch more scoped exception, instead of all of them.